src/app/_pages/join/join.page.ts

```diff
@@ -1,21 +1,69 @@
 import { Component } from '@angular/core';
 import { ApiService } from '_services/api.service';
+import { Group } from '_models/group.model';
 
 @Component({
   selector: 'join-page',
   templateUrl: 'join.page.html',
   styleUrls: ['join.page.scss'],
 })
 export class JoinPage {
-  public housingGroups: string[] = ['Pittsburgh', 'Oakland', 'Shadyside', 'Squirrel Hill', 'Lawrenceville', 'Southside', 'East Liberty', 'Strip District', 'Mount Washington', 'Downtown', 'Duquesne Heights'];
+  public joinGroups: Group[] = [];
+  public selectedOptions: Group[] = [];
+  public userAccount: any;
+  public currentGroups: any = [];
+
   constructor(
     private apiService: ApiService,
   ) {
-
   }
 
   ngOnInit() {
+    this.getAccount();
+  }
 
+  joinSelectedGroups() {
+    for (var group of this.selectedOptions) {
+      let accountGroup = {
+        "GroupId": group.id,
+        "AccountId": this.userAccount.id
+      }
+      this.apiService.createAccountGroup(group.id, accountGroup).subscribe(res => {
+        this.getJoinGroups();
+        setTimeout(function(){ // Refresh page so joined group shows up on side bar automatically.
```

Comment:
commented in slack. this should be removed and switched to some sort of event or observable to trigger the reload of data in the sidebar.

Reply:
Fixed!